File: NetFlexAPI/NetFlexAPI/Models/Film.cs
namespace NetFlexAPI.Models;

public class Film
{
    public Guid Id { get; set; }
    public string Title { get; set; }
    public string Poster { get; set; }
    public int AgeRating { get; set; }
    public float UserRating { get; set; }
    public string Description { get; set; }
    public string VideoLink { get; set; }
}

File: NetFlexAPI/GiraffeAPI/Models/Film.fs
module GiraffeAPI.Models.Film

open System
open System.ComponentModel.DataAnnotations
open Microsoft.EntityFrameworkCore.Metadata.Internal

[<CLIMutable>]  
type Film =
    {
        Id : Guid
        Title : string
        Poster : string
        AgeRating : int
        UserRating : float
        Description : string
        VideoLink : string
    }
    
    
    